src/app/views/marketplace/marketplace-app.controller.coffee

```diff
@@ -41,6 +41,14 @@ angular.module 'mnoEnterpriseAngular'
 
         vm.isLoading = false
 
+      # Update the comments and avergae review
+      vm.updateReview = (newReview) ->
+        # Select and push to comments the last comment added
+        [first, ..., last] = newReview.app.reviews
```

Comment:
Please review variable name: `newReview.app.reviews` is confusing

I guess it's `response.app.reviews`